### ISSUE TITLE
Plugin settings buttons are not accessible through keyboard navigation

### ISSUE THREAD

**Issue by haykam821** [badasintended/wthit#196]
### Describe the issue

All buttons should be navigable via the <kbd>Tab</kbd> and arrow keys. However, the buttons found in the Plugin Settings screen, which are 'Waila Core' and 'Minecraft' by default, cannot be navigated to through these methods.

### Log output and crash report

Minecraft 1.19.4 release candidate 2, Fabric Loader 0.14.17, Fabric API 0.75.3, WTHIT 7.0.0

### Additional context

This issue also exists in previous versions of WTHIT for Minecraft 1.19.3.
